src/sentry/south_migrations/0418_auto__del_grouptagvalue__del_unique_grouptagvalue_group_id_key_value__.py

```diff
@@ -0,0 +1,1127 @@
+# -*- coding: utf-8 -*-
```

Comment:
So I verified I was based off latest master, and I dropped and recreated my `sentry` tables... but the auto generated migration was full of garbage (like dropping a lot of indexes). I'm going to assume the previously serialized South state is out of sync? I edited down to the new tables I want to add. Is there something I/we should do to fix the state, though?